cmd/webhook/main.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/awslabs/karpenter/pkg/apis"
 	"github.com/awslabs/karpenter/pkg/cloudprovider"
 	"github.com/awslabs/karpenter/pkg/cloudprovider/registry"
+	karpenterinjection "github.com/awslabs/karpenter/pkg/utils/injection"
```

Comment:
consider injection for karpenter and kinjection or knativeinjection for knative

Reply:
done